compiler/crates/relay-compiler/tests/compile_relay_artifacts/fixtures/relay-resolver-edge-to-interface-with-no-implementors.expected

```diff
@@ -0,0 +1,38 @@
+==================================== INPUT ====================================
+# expected-to-throw
+query relayResolverEdgeToInterfaceWithNoImplementorsQuery {
+  resolver_field {
+    name
+  }
+}
+
+# %extensions%
+
+"""
+An interface with no implementors
+"""
+interface SomeInterface {
+  name: String
+}
+
+extend type Query {
+  resolver_field: SomeInterface
+    @relay_resolver(import_path: "./path/to/Resolver.js")
+}
+==================================== ERROR ====================================
+✖︎ Client Edges that reference client-defined interface types are not currently supported in Relay.
+
+  relay-resolver-edge-to-interface-with-no-implementors.graphql:3:3
+    2 │ query relayResolverEdgeToInterfaceWithNoImplementorsQuery {
+    3 │   resolver_field {
+      │   ^^^^^^^^^^^^^^
+    4 │     name
+
+
+✖︎ No types implement the client interface SomeInterface. For a client interface to be used as a @RelayResolver @outputType, at least one Object type must implement the interface.
+
+  <generated>:2:35
+    1 │ # expected-to-throw
+    2 │ query relayResolverEdgeToInterfaceWithNoImplementorsQuery {
+      │                                   ^^^^^^^^^^^^^
```

Comment:
This underline being in the wrong place is not a problem with your diff. Our test util is not fully roust in this case and does not print location info correctly.

Reply:
Right, it should be underlining `resolver_field` here. Just wondering, where does that get determined?

Reply:
Here: https://github.com/facebook/relay/blob/74bb8abd92d13460fb90be290c65719cba74c62e/compiler/crates/relay-compiler/tests/compile_relay_artifacts/mod.rs#L206

The problem is that the fixture file actually models two different files, and so the error location should actually get mapped to an offset based on if its an error in the schema portion of the file or the fragment portion.